### ISSUE TITLE
All Shapes - Type 'xxx' is missing the following properties from type 

### ISSUE THREAD

**Issue by OldManMeta** [pmndrs/drei#1343]
Having a lot of issues with drei specific components at the moment. In particular the shapes

Several of them keep showing in my solution as types missing properties.

I've cleaned out node_modules, rebooted the whole 9 yards - to no avail.

I have checked on codesandbox and it appears to work correctly with the same versions that I am using (latest of r3f, drei etc)

I am almost certain this is a local issue of mine, but just in case anyone else is seeing this with the current build?

Is there any resolution that I can apply to fix this, or steps that folks have taken in the past to counter it as I saw a while back that it was an issue, but I was not able to ascertain the fix.

"dependencies": {
    "@aws-sdk/client-s3": "^3.282.0",
    "@emotion/react": "11.10.4",
    "@emotion/styled": "11.10.4",
    "@mui/icons-material": "5.10.3",
    "@mui/lab": "5.0.0-alpha.96",
    "@mui/material": "5.10.3",
    "@react-spring/web": "^9.6.1",
    "@react-three/drei": "^9.57.3",
    "@react-three/fiber": "^8.12.0",
    "@react-three/rapier": "^0.13.2",
    "@types/react": "18.0.18",
    "@types/react-dom": "18.0.6",
    "@types/three": "^0.146.0",
    "electron": "^18.0.3",
    "framer-motion": "^6.3.0",
    "leva": "0.9.34",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-hook-form": "^7.29.0",
    "react-query": "^3.38.0",
    "react-router-dom": "^6.3.0",
    "react-use-websocket": "^4.3.1",
    "rxjs": "7.5.6",
    "three": "^0.147.0",
    "three-stdlib": "^2.21.8"
  },
  "devDependencies": {
    "css-loader": "^6.7.1",
    "electron-packager": "^15.5.1",
    "file-loader": "^6.2.0",
    "filemanager-webpack-plugin": "^7.0.0",
    "html-webpack-plugin": "^5.5.0",
    "source-map-loader": "^3.0.1",
    "ts-loader": "^9.2.8",
    "typescript": "^4.6.3",
    "webpack": "^5.74.0",
    "webpack-cli": "^4.9.2",
    "webpack-dev-server": "^4.11.1"
  }
}

**Comment by AlexanderMoroz** [comment]
Faced same issue after updating to latest `@react-three/fiber` and `@react-three/drei`

currently 
```
    "@react-three/drei": "^9.57.3",
    "@react-three/fiber": "^8.12.0",
```
it was working on well with:
```
    "@react-three/drei": "^9.56.4",
    "@react-three/fiber": "^8.10.1",
 ```
 
Thanks in advance

**Comment by queckezz** [comment]
@OldManMeta My types where fixed by updating from `@types/three@0.147.0` (like you have) to `@types/three@0.150.1`. Maybe that'll help you aswell
